src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,96 @@
+package duke;
+
+import java.util.List;
+import java.util.Scanner;
+
+import duke.commands.CommandExecution;
+import duke.commands.EnumCommand;
+import duke.exception.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.tasks.Task;
+import duke.tasks.TaskList;
+import duke.ui.Ui;
+
+/**
+ * A class that represents the Duke application which contains the main method in the class.
+ */
+public class Duke {
+    private static final String filePath = "duke.txt";
+    private Storage storage;
+    private TaskList results;
+    private Ui ui;
+
+    /**
+     * Sets up the user interface and load list from file storage.
+     *
+     * @param filePath the path of the file storage of list of tasks.
+     */
+    public Duke(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            results = new TaskList();
+            results = new TaskList();
+            List<Task> taskList = storage.readFromFile();
+            taskList.stream().forEach(task -> results.add(task));
+        } catch (DukeException e) {
+            ui.showError(e);
+            results = new TaskList();
+        }
+    }
+
+    /**
+     * Constructs a Duke object without taking in arguments.
+     */
+    public Duke() {
+
+    }
```

Comment:
There is no need to declare an empty constructor method. When there is no constructor declared, you can just use new duke() and it will automatically create a duke object with no variables.